charger/em2go.go

```diff
@@ -158,8 +176,32 @@ func (wb *Em2Go) Enable(enable bool) error {
 	b := make([]byte, 2)
 	binary.BigEndian.PutUint16(b, map[bool]uint16{true: 1, false: 2}[enable])
 
-	_, err := wb.conn.WriteMultipleRegisters(em2GoRegChargeCommand, 1, b)
-	return err
+	if _, err := wb.conn.WriteMultipleRegisters(em2GoRegChargeCommand, 1, b); err != nil {
+		return err
+	}
+
+	// Workaround
+	if wb.workaround {
```

Comment:
Warum brauchts das hier? Genügt es nicht, das nur in der Phasenumschaltung zu machen?

Reply:
Ich denke es wird auch bei Enable benötigt, da sonst der Charger mit der alten FW by default mit 3p startet, obwohl mit 1p gestartet werden soll. Wenn wir dies streichen, würde der Charger mit 3p starten und erst auf 1p springen, nachdem evcc dies bemerkt. Hier kann einige Zeit verstreichen in der mit 3p geladen wird, was vom Benutzer nicht gewollt ist. 

Wie zuvor kommuniziert, werden Phaseneinstellungen vor Enable leider nicht vom Charger übernommen, daher ist dies notwendig. Wird aber nur bei alter FW angewendet.